src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -105,11 +105,6 @@ public Task InitializeAsync(ProtectedPrerenderComponentApplicationStore store, C
                     await OnCircuitOpenedAsync(cancellationToken);
                     await OnConnectionUpAsync(cancellationToken);
 
-                    // From this point onwards, JavaScript code can add root components if configured
-                    await Renderer.InitializeJSComponentSupportAsync(
-                        _options.RootComponents.JSComponents,
-                        JSRuntime.ReadJsonSerializerOptions());
```

Comment:
This kind of initialization is now done from the `WebRenderer` constructor instead, so it's common to all.